src/dune

```diff
@@ -7,4 +7,4 @@
    ppx_tools)
  (modes js)
  (js_of_ocaml
-  (flags --source-map --pretty)))
+  (flags --source-map)))
```

Comment:
Why this change?

Reply:
Sorry, this was a confusion after upgrading to dune 3. I reverted that already. 
Related: https://github.com/ocaml/dune/issues/5563